src/transforms/log_to_metric.rs

```diff
@@ -191,11 +206,25 @@ enum TransformError {
     FieldNull {
         field: String,
```

Comment:
Let's rename all `field` fields in the `TransformError` enum to `path`.

Reply:
Done.